src/GitHub.VisualStudio/Settings/generated/PackageSettingsGen.cs

```diff
@@ -43,6 +43,13 @@ public bool CollectMetrics
             set { collectMetrics  = value; this.RaisePropertyChange(); }
         }
 
+        bool enableTraceLogging;
```

Comment:
Again, this is a generated file so shouldn't be editing it directly.

Reply:
Cool